src/query.js

```diff
@@ -0,0 +1,60 @@
+const soymineQuery = geneId => ({
+	from: 'GWAS',
+	select: [
+		'results.study.primaryIdentifier',
+		'results.phenotype.primaryIdentifier',
+		'results.marker.primaryIdentifier',
+		'results.pValue',
+		'results.marker.chromosome.secondaryIdentifier',
+		'results.marker.chromosomeLocation.start',
+		'results.marker.chromosome.length'
+	],
+	joins: ['results.study', 'results.phenotype', 'results.marker'],
+	where: [
+		{
+			path: 'id',
+			op: 'ONE OF',
+			values: geneId
+		}
+	]
+});
+
+const humanMineQuery = geneId => ({
+	from: 'GWAS',
+	select: [
+		'results.pValue',
+		'results.phenotype',
+		'results.associatedGenes.chromosomeLocation.start',
+		'results.associatedGenes.chromosome.length',
+		'results.associatedGenes.chromosome.primaryIdentifier',
+		'results.associatedGenes.primaryIdentifier'
+	],
+	where: [
+		{
+			path: 'results.associatedGenes.id',
+			op: 'ONE OF',
+			values: geneId
```

Comment:
Sorry, I probably explained myself poorly during the call!

We will drop the idea of matching genes, so this constraint should be identical to the Soymine query:
```
	where: [
		{
			path: 'id',
			op: 'ONE OF',
			values: geneId
		}
	]
```